nixos/release-small.nix

```diff
@@ -4,7 +4,7 @@
 
 { nixpkgs ? { outPath = (import ../lib).cleanSource ./..; revCount = 56789; shortRev = "gfedcba"; }
 , stableBranch ? false
-, supportedSystems ? [ "x86_64-linux" ] # no i686-linux
+, supportedSystems ? [ "x86_64-linux" "aarch64-linux" ] # no i686-linux
```

Comment:
Nit: When rewording this commit also name it `smoke test`

Reply:
Ok, I intend to drop that commit before rebasing onto staging anyway. I'll fix the name if we need another revision of these patches.